src/components/whiteboard.js

```diff
@@ -6,10 +6,13 @@ import "./whiteboard.scss";
 import { roomStore } from "../stores/room";
 import { fileContext } from "./mediaboard";
 import { toggleNext, togglePrev, toggleFirstLast } from "./whiteboard/control";
+import FullScreen from './fullscreen/index';
+
 import FullscreenOutlinedIcon from '@material-ui/icons/FullscreenOutlined';
```

Comment:
Is this required to import here because you have already imported in fullscreen/index.js

Reply:
Not required, i will remove this line.